src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -274,7 +266,7 @@ tests:
           [1670970505.890328][5742:5744] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0005 Command 0x0000_0040
           [1670970505.890458][5742:5744] CHIP:TOO:   EnhancedAddSceneResponse: {
           [1670970505.890551][5742:5744] CHIP:TOO:     status: 0
-          [1670970505.890608][5742:5744] CHIP:TOO:     groupId: 1
+          [1670970505.890608][5742:5744] CHIP:TOO:     groupId: 101
```

Comment:
It's 0x101 in the header above.  Will it really get logged as "101" here instead of 257?  Same below.

Reply:
257